service/git/syncer.go

```diff
@@ -53,6 +54,8 @@ func (s *Syncer) Sync(ctx context.Context,
 }
 
 func (s *Syncer) fetch(ctx context.Context, repoURL string, r *git.Repository, refspecs []config.RefSpec) error {
+	log.Infof("fetching references for repository %s: %v", repoURL, refspecs)
```

Comment:
As this process might take some time, do you think it might make sense to add another log at the end of fetching?

Reply:
example of the logs
```
$ make dry-run
go run cmd/lookout/*.go serve --dry-run github.com/src-d/lookout
[2018-08-09T16:08:24.187459705+02:00]  INFO Starting watcher urls=[github.com/src-d/lookout]
[2018-08-09T16:08:24.190722352+02:00]  INFO connection state changed to 'READY' addr=ipv4://localhost:10302 analyzer=Dummy
[2018-08-09T16:08:27.589335009+02:00]  INFO event successfully processed, skipping... event-id=a29ea85b92358e61e3a56281c5f6e885fa170ab7 event-type=2
[2018-08-09T16:08:27.7994484+02:00]  INFO processing push app=lookout head=refs/heads/master provider=github repository=git://github.com/src-d/lookout.git
[2018-08-09T16:08:27.799639533+02:00]  INFO creating local repository for: git://github.com/src-d/lookout.git app=lookout
[2018-08-09T16:08:27.801553032+02:00]  INFO fetching references for repository git://github.com/src-d/lookout.git: [refs/heads/master:refs/heads/master] app=lookout
[2018-08-09T16:08:39.005883394+02:00]  INFO repository config is not found app=lookout head=refs/heads/master provider=github repository=git://github.com/src-d/lookout.git
[2018-08-09T16:08:39.005989076+02:00]  INFO status: pending app=lookout
[2018-08-09T16:08:39.006093794+02:00]  INFO unary client call started app=lookout grpc.method=NotifyPushEvent grpc.service=pb.Analyzer span.kind=client system=grpc
```
you can clearly see when it finished fetching. Imo that is more than enough for logs.
Or do you want to know how much time did it take? Should it be in the logs or in tracing?

Reply:
> Or do you want to know how much time did it take?

yes, was thinking that could be easy way to see elapsed time, until we have a tracing